Fix build failure caused by Monoid.<> clash

[2 of 8] Compiling Network.XmlRpc.Pretty ( Network/XmlRpc/Pretty.hs, dist/build/Network/XmlRpc/Pretty.o )

Network/XmlRpc/Pretty.hs:51:10:
    Ambiguous occurrence `<>'
    It could refer to either `Network.XmlRpc.Pretty.<>',
                             defined at Network/XmlRpc/Pretty.hs:57:1
                          or `Data.Monoid.<>',
                             imported from `Data.Monoid' at Network/XmlRpc/Pretty.hs:20:1-18
diff --git a/Network/XmlRpc/Pretty.hs b/Network/XmlRpc/Pretty.hs
index bd9397b..2590558 100644
--- a/Network/XmlRpc/Pretty.hs
+++ b/Network/XmlRpc/Pretty.hs
@@ -17,7 +17,7 @@ import Text.XML.HaXml.Types
 import Blaze.ByteString.Builder (Builder, fromLazyByteString, toLazyByteString)
 import Blaze.ByteString.Builder.Char.Utf8 (fromString)
 import Data.Maybe (isNothing)
-import Data.Monoid
+import Data.Monoid (mappend, mempty, mconcat, Monoid(..)) -- hide Monoid.<>
 import qualified GHC.Exts as Ext
 
 -- |A 'Builder' with a recognizable empty value.
